vignettes/tic.Rmd

```diff
@@ -276,4 +230,4 @@ The advanced usage of *tic* is described in more detail in the [Advanced Usage](
 - [Troubleshooting: Enter into the Travis build](advanced.html#troubleshooting-running-tic-locally)
 - [Writing custom steps](custom-steps.html)
 
-The build lifecycle when using *tic* is explained in the [Build lifecycle](build-lifecycle.html) article.
+For more information on the build lifecycle see [here](build-lifecycle.html).
```

Comment:
```suggestion
For more information on the build lifecycle see the ["Build lifecycle" article](build-lifecycle.html).
```

I remember reading somewhere that links titled "here" aren't recommended. When looking for the source I've found https://silktide.com/blog/2013/i-thought-title-text-improved-accessibility-i-was-wrong which is interesting in itself. I'm not sure the reasons against "weak link text" stated there are the only ones, but these are important already.

Reply:
Sounds interesting. Thanks for sharing.